doc/plugins.md

```diff
@@ -182,6 +182,8 @@ The list of plugins:
 * [`rehype-sectionize`](https://github.com/hbsnow/rehype-sectionize)
   — wrap headings and their contents into nested `<section>` elements,
   with attributes
+* [`rehype-semantic-blockquotes`](https://github.com/nikitarevenco/rehype-semantic-blockquotes)
+  — extend blockquote syntax to make it simple to mention/cite sources in a semantically correct way
```

Comment:
```suggestion
* [`rehype-semantic-blockquotes`](https://github.com/nikitarevenco/rehype-semantic-blockquotes)
  — new blockquote syntax to more simply mention/cite sources
```

How about this? A bit shorter.

If you think the semantically correct part is important, you could add it. Just make sure it line wraps correctly (CI tells you about problems: `npm install && npm test`!)

Reply:
Yes, thank you! I have slightly reworded it to make sure it's under 80 chars with inspiration from your suggestion